Barotrauma/BarotraumaClient/ClientSource/GameSession/CrewManager.cs

```diff
@@ -1373,7 +1374,8 @@ private int TryAdjustIndex(int amount)
                 if (index > lastIndex) { index = 0; }
                 if (index < 0) { index = lastIndex; }
 
-                if ((crewList.Content.GetChild(index)?.UserData as Character)?.IsOnPlayerTeam ?? false)
+                var candidateCharacter = crewList.Content.GetChild(index)?.UserData as Character;
+                if (candidateCharacter != null && !candidateCharacter.Removed && candidateCharacter.IsOnPlayerTeam)
```

Comment:
This can be done a bit more cleanly with pattern matching:
`crewList.Content.GetChild(index)?.UserData is Character { IsOnPlayerTeam: true, Removed: false }`